PoW-vs-Pos/en/en_transcription.md

```diff
@@ -0,0 +1,102 @@
+
+Transcript:
+
+(00:02) in the winter of 2009 a piece of software was released on the internet digital money called bitcoin like throwing water on burning oil it ignited and this is the biggest story of this decade bitcoin sent shockwaves through the world spreading like wildfire and disrupting everything it touched nobody knew what it was nobody knew how to value it and nobody could control it and like any new technology it came with pushback the whole bitcoin market is about a billion dollars it's a novelty for the geek squad on top of all
+
+(00:34) of these attacks critics came after its energy usage that's right bitcoin consumes energy bitcoin in general is wasteful by design bitcoin uses a system called proof of work which requires energy to function bitcoin uses more energy than countries like ukraine norway sweden argentina or the philippines now if you just heard of those comparisons you're likely shocked and here's where proof of stake comes in i think great things are on the other side so number one is obviously proof of stake so much more
+
+(01:09) efficient much more secure much less rekilling thousands of new cryptocurrencies use this new proof of stake model all promising to be superior to the old energy intensive proof-of-work technology there's only one problem is this true or are they marketing false promises one of my issues with sort of proof of stake the way it's been presented and with the ethereum roadmap is that i think that the risks are not sort of fully discussed in a very open intellectually honest way from this video we will see
+
+(01:39) the differences between proof of work and proof of stake the problems tied to each and whether one is more efficient than the other often the advantages of proof of stake are described without acknowledgement of its trade-offs compared to proof of work to understand these trade-offs we must first break down what proof of work is the bitcoin network is programmed to create a new block every 10 minutes and add that block to the blockchain a new block is produced by a bitcoin miner contributing processing power to solve a
+
+(02:09) cryptographic puzzle that the previous block created at this point the miner can package thousands of bitcoin transactions into that block that's how transactions get settled the youtube channel three blue one brown explains the other half of proof of work in this short clip earlier i said that the proof of work might be to find a special number so that the hash of the block starts with 60 zeros well the way the actual bitcoin protocol works is to periodically change that number of zeros so that it should take
```

Comment:
Distinguere le frasi non del narratore `earlier i said that the proof of work might be to find a special number so that the hash of the block starts with 60 zeros well [...] and harder in such a way that this miniature lottery only has about one winner every 10 minutes`